pkg/scheduler/apis/config/types.go

```diff
@@ -210,6 +229,217 @@ type PluginConfig struct {
 	Args runtime.Unknown
 }
 
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
```

Comment:
should we place those in a separate legacy_types.go file? this will be useful to clarify that this is an older interface.

Reply:
Sounds good, although I haven't seen this pattern before :)

Let me see if the naming is ok with our CI boss.